notebooks/python/python-basics.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>Consider saying "assign the value of an expression to a variable" instead of "assign an expression to a variable" in the markdown cell about assigning variables.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ESCIP/escip.github.io/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>